lib/pure/sugar.nim

```diff
@@ -335,6 +335,20 @@ macro collect*(init, body: untyped): untyped {.since: (1, 1).} =
       call.add init[i]
   result = newTree(nnkStmtListExpr, newVarStmt(res, call), resBody, res)
 
+since (1, 3):
+  template c*(s: string{nkRStrLit}): cstring =
+    ## Convenience template for `cstring("some string")`, *only for Literal* `string`.
+    ## Wont match `"str".c` nor `c("str")`, just `c"str"`, to allow functions named `c()`.
+    ## Meant for working with JavaScript or C or C++ interoperability with lots of `cstring`.
+    runnableExamples:
+      static:
+        proc c(s: string) = doAssert false, "Must not match this"
+        doAssert c"Nim" == cstring("Nim")
+        doAssert c" " == cstring(" ")
+        doAssert c"" == cstring("")
+    system.cstring(s)
```

Comment:
```suggestion
    bind cstring
    cstring(s)
```

Explicit `bind()` is usually better as the symbol will be resolved directly at definition site.